### ISSUE TITLE
Upon reflow, the "overflow" Tab button is not accessible through keyboard key

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/fluentui-blazor#3721]
**Githubs Tags**:  #A11ySev1; #DesktopWeb; #A11yMAS; #A11yTCS; #Win11; #WCAG2.1.1; #FTP; #ChromiumEdge;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility after verification of the bug".**

## Environment Details:
Application Name: Fluent UI Blazor
Page URL: https://www.fluentui-blazor.net/Tabs
Chromium Edge Version 135.0.3179.73 (Official build) (64-bit)

## Repro Steps:

1. Launch #URL: https://www.fluentui-blazor.net/Tabs
2. Apply Reflow.
3. Tab to the section where we have overflow Tab button.
4. Observe if the element is accessible.

## Actual Result:

The "overflow" button is not accessible via keyboard navigation after the reflow.

## Expected Results:

The "overflow" button should be accessible through keyboard navigation after the reflow.

## User Impact:

Users who depend on keyboard navigation are unable to access the "overflow" button, blocking them from accessing the elements present in the expanded section of "overflow" button.

## WCAG Reference:

https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

**Comment by MarvinKlein1508** [comment]
I can reach them by keyboard by pressing the arrow buttons:
![Image](https://github.com/user-attachments/assets/739cf5c4-ac99-4fb8-9ba5-10f51df7d260)

**Comment by ManishaC1** [comment]
@MarvinKlein1508 
Yes, the contents of the tab can be accessed using the arrow keys. However, the user may not be aware of which tab section they are currently in. By making the overflow tab accessible through keyboard keys, users can see all the tabs included in the overflow section and select the desired one.

**Comment by MarvinKlein1508** [comment]
@ManishaC1 how should they be aware if the tab is hidden in the overflow menu?

I've tried a few things here. Adding  `tabindex="0"` to the `FluentSpan` allows the other to tab to it, however he cannot open the menu via keyboard then. Exchanging the `FluentSpan` with a `FluentButton` fixes this issue. However the user then cannot tab/navigate through the `FluentMenu`.

And even then my question remains: How should the user know which tab is currently active from the overflow menu? As you can see in my GIF above even the indicator gets confused.

**Comment by vnbaaij** [comment]
As @MarvinKlein1508 mentioned, even the indicator gets confused. This is due to the underlying web component not working well with the dynamic tabs. The dynamic tabs is something we added on top of the web component. We cannot change nor fix the web component in the version we are using here (v2).** So for v4 of the library this will remain unfixed.**

We will work with the web component team for the new tab component (v3) to handle this scenario in an accessible way in our next version of the library (v5)

**Comment by vnbaaij** [comment]
@ManishaC1 please respond if this needs to stay open (blocked) or if we can close.

**Comment by ManishaC1** [comment]
@vnbaaij 
Please keep the bug open, as the issue is still unresolved.
And the partner team is asking about the release timeline for v5. Could you please confirm the expected release date

**Comment by vnbaaij** [comment]
I'm closing this as it ill not be fixed for v4. It is not possible to change/fix this with the current underlying web component/overflow combination,
